docs/dev-setup.md

```diff
@@ -76,6 +76,14 @@ The frontend is run with `yarn`.  We recommend running the backend via `eclipse`
 1. Run `./gradlew build` to download all dependencies and build the project.
    - Some tests will fail if you are using SQLite, because it has a bad implementation of some timezone stuff. You'll need to use MSSQL to see all the tests passing.
 
+#### Initial Setup Test Database
```

Comment:
Describe the possibility to override the `build.gradle` test defaults by using a `localTestSettings.gradle`.
It is probably also worth adding `localTestSettings.gradle` to the top-level `.gitignore`.